src/Win32/Win32.csproj

```diff
@@ -7,7 +7,15 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Microsoft.CodeAnalysis.Analyzers" Version="2.9.5-beta1.final">
```

Comment:
These are still/already set in build/NetStandardRelease.targets. I'm wondering if the analyzers should be in NetStandardAll.targets

Reply:
Per our conversation, removed from NetStandardRelease.targets and leaving them where VS tooling can update them